### DISCUSSION TITLE
Doubt about FSK mode decode

### DISCUSSION THREAD

**Discussion by jp-polito** [jgromes/RadioLib#564] [Q&A]
Hello Guys!
Has anyone tried decoding FSK using SoundModem? I'm using the standard RadioLib example but I'm not having success in decoding the data :(
Thanks!

**Comment by jgromes** [comment]
As far as I know, SoundModem works with AFSK? You will have to provide more details about the code you're running and your reception setup.

Also, this is not really an issue - moving into Discussions instead.
